__tests__/app.test.js

```diff
@@ -342,4 +342,65 @@ describe("GET /api/users", () => {
             })
         })
     })
+})
+
+describe("GET /api/users (queries)", () => {
```

Comment:
Did you copy and paste the description from earlier 🤔 😂

Reply:
maybe hehehehehehe